package.json

```diff
@@ -20,7 +20,7 @@
     "compile-changed": "lerna run compile --stream --since origin/HEAD",
     "depalign": "node scripts/depalign.js",
     "predepcheck": "npm run depalign",
-    "depcheck": "depcheck",
+    "depcheck": "depcheck && lerna run depcheck --stream",
```

Comment:
This will make depcheck run twice in all the packages in ci as all the `check` scripts in workspaces are alreay set up to run depcheck as part of the template

Reply:
ah cause the `npm run depcheck` is also called in ci?

Reply:
For the root it's both happening in CI and locally with the "pre*" script:

<img width="597" alt="image" src="https://user-images.githubusercontent.com/5036933/144420811-be6ca5f7-2f84-4ad1-a735-398badc7f4a1.png">

Reply:
ok thanks!